stackslib/src/burnchains/tests/mod.rs

```diff
@@ -435,6 +456,10 @@ impl TestBurnchainBlock {
         )
         .expect("FATAL: failed to read block commit");
 
+        if parent_is_shadow {
+            assert!(get_commit_res.is_none());
```

Comment:
Nit: would be good probably to add a message here for the failure case like "Parent block is a shadow block. There should be no block commit." or something like that.

Reply:
Done